examples/sktime/README.md

```diff
@@ -0,0 +1,31 @@
+# Sktime Example
+
+This example trains a Sktime NaiveForecaster model using the Longley dataset for
+forecasting with exogenous variables and logs hyperparameters, metrics, and
+trained model.
+
+## Running the code
+
```

Comment:
Intro here? Something that explains the code below would be helpful as a guide

Reply:
updated